src/scripts/Controller.js

```diff
@@ -586,6 +589,15 @@ export default class SitemapController {
 		return true;
 	}
 
+	copyText() {
```

Comment:
Consider using more elaborate names here, e.g. `copySitemapToClipboard`. 
It would help with readability, it is not clear from `copyText` what is being copied and where.
Also I'd suggest `copyTextArea` -> `sitemap-area`.